tests/system/python/conftest.py

```diff
@@ -212,6 +215,138 @@ def clone_make_install():
+            retrigger_time: Interval between retriggered notifications
+            use_pip_cache: use pip cache in requirements installation
+            enabled: Flag to enable or disable notification instace
+        """
+        PROJECT_ROOT = Path(__file__).parent.parent.parent.parent
+        
+        if rule_plugin_discovery_name is None:
+            rule_plugin_discovery_name = rule_plugin
+            
+        if delivery_plugin_discovery_name is None:
+            delivery_plugin_discovery_name = delivery_plugin
+
+        def clone_make_install(plugin_branch, plugin_type, plugin):
+            try:
+                subprocess.run(["{}/tests/system/python/scripts/install_c_plugin {} {} {}".format(
+                    PROJECT_ROOT, plugin_branch, plugin_type, plugin)], shell=True, check=True)
+            except subprocess.CalledProcessError:
+                assert False, "{} plugin installation failed".format(plugin)
+
+        if installation_type == 'make':
+            # Install Rule Plugin if it is not Threshold or DataAvailability
+            if rule_plugin not in  ("Threshold","DataAvailability"):
+                clone_make_install(rule_branch, "rule", rule_plugin)
+            
+            clone_make_install(delivery_branch, "notify", delivery_plugin)
+            
+        elif installation_type == 'package':
+            try:
+                if rule_plugin not in ["Threshold", "DataAvailability"]:
+                    subprocess.run(["sudo {} install -y fledge-rule-{}".format(pytest.PKG_MGR, rule_plugin)], shell=True,
+                               check=True)
+                    
+            except subprocess.CalledProcessError:
+                assert False, "Package installation of {} failed!".format(rule_plugin)
+                
+            try :
+                subprocess.run(["sudo {} install -y fledge-notify-{}".format(pytest.PKG_MGR, delivery_plugin)], shell=True,
+                               check=True)
+                
+            except subprocess.CalledProcessError:
+                assert False, "Package installation of {} failed!".format(delivery_plugin)
+        else:
+            return("Skipped {} and {} plugin installation. Installation mechanism is set to {}.".format(rule_plugin, delivery_plugin,
+                                                                                                installation_type))
+
+        data = {
+                "name": notification_instance_name,
+                "description": "{} notification instance".format(notification_instance_name),
+                "rule_config": rule_config,
+                "rule": rule_plugin_discovery_name,
+                "delivery_config": delivery_config,
+                "channel": delivery_plugin_discovery_name,
+                "notification_type": notification_type,
+                "enabled": enabled, 
+                "retrigger_time": "{}".format(retrigger_time),
```

Comment:
Can be without format

Reply:
I tried it without format but it configuration is not updated in fledge. That's why I used format.

Reply:
I got it as API expects every KV pair in string that's why it is working with format.
Either `str(retrigger_time) or "{}".format(retrigger_time)` both will work.